examples/nima/imperative/src/main/java/io/helidon/examples/nima/imperative/ImperativeMain.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.nima.imperative;
+
+import io.helidon.common.config.Config;
+import io.helidon.common.http.Http;
+import io.helidon.nima.Nima;
+import io.helidon.nima.webserver.WebServer;
+import io.helidon.nima.webserver.http.HttpRouting;
+
+/**
+ * Main class of this example, starts the server.
+ */
+public class ImperativeMain {
```

Comment:
Seems to reason we should also have a DeclarativeMain? That would drive pico `RunLevel` based services perhaps, as well as seeding config bootstrap, etc.

Reply:
All other examples will be based on declarative approach, eventually